setup-buildengine.sh

```diff
@@ -172,6 +173,15 @@ installTarget() {
         mersdk@localhost "sdk-manage --mode installer --target --install --jfdi $tgt file:///home/mersdk/share/$TARGET_FILENAME"
 }
 
+createTar7z() {
+    echo "Compressing filesystem ..."
+    ssh -o UserKnownHostsFile=/dev/null \
+        -o StrictHostKeyChecking=no \
+        -p $SSH_PORT \
+        -i $SSHCONFIG_PATH/vmshare/ssh/private_keys/engine/mersdk \
+        mersdk@localhost "sudo tar -cf - --one-file-system / | 7z a -si /home/mersdk/share/mer.tar.7z"
```

Comment:
We might do better with the `--numeric-owner` option to tar here.

Reply:
Maybe so. I'll add that.